### PR TITLE
update getting started guide

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

File: app/views/docs/getting-started-for-server.phtml [hunk 1]
```diff
@@ -85,7 +86,7 @@ $kotlinVersion = $versions['kotlin'] ?? '';
             <div class="ide margin-top-small" data-lang="swift" data-lang-label="Swift">
             <pre class="line-numbers"><code class="prism language-swift" data-prism>dependencies: [
     // ...
-    .package(url: "https://github.com/appwrite/sdk-for-swift.git", from: "0.0.1")
+    .package(url: "https://github.com/appwrite/sdk-for-swift.git", from: "?php $this->escape($swiftVersion); ?>")
```
Comment (Meldiron) [inline app/views/docs/getting-started-for-server.phtml:89]:
```suggestion
    .package(url: "https://github.com/appwrite/sdk-for-swift.git", from: "<?php $this->escape($swiftVersion); ?>")
```

File: app/views/docs/getting-started-for-server.phtml [hunk 2]
```diff
@@ -318,7 +319,6 @@ $client
         <h3>Python</h3>
         <div class="ide margin-top-small" data-lang="python" data-lang-label="Python SDK">
             <pre class="line-numbers"><code class="prism language-python" data-prism>from appwrite.client import Client
-from appwrite.services.users import Users
```
Comment (Meldiron) [inline app/views/docs/getting-started-for-server.phtml:321]: Was this deletion on purpose?

**Review by Meldiron — CHANGES_REQUESTED** [review]

File: app/views/docs/getting-started-for-server.phtml (hunk 2 above)
Comment (lohanidamodar) [reply app/views/docs/getting-started-for-server.phtml:321]: yes, this is SDK init section, no need to import users here

Comment (Meldiron) [reply app/views/docs/getting-started-for-server.phtml:321]: Okay

**Review by Meldiron — APPROVED** [review]
LGTM

**Review by eldadfux — APPROVED** [review]

**Pull request merged**